### PR TITLE
Adding IngeoML

### PR DESCRIPTION
**Pull request by mgraffg**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mgraffg** [comment]
@conda-forge-admin, please restart ci

**Comment by mgraffg** [comment]
@conda-forge-admin, please restart ci

File: recipes/ingeoml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - scikit-learn
```
Comment (synapticarbors) [inline recipes/ingeoml/meta.yaml:20]: `scikit-learn` is unlikely to be needed at build time

File: recipes/ingeoml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - scikit-learn
```
Comment (synapticarbors) [inline recipes/ingeoml/meta.yaml:22]: Upstream requires `scikit-learn >=1.3.0`

File: recipes/ingeoml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: 38bcd14a51fa2579e8db81da5bf5bb13f3f53b69c35a59bc47a7d62a7c939c5c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (synapticarbors) [inline recipes/ingeoml/meta.yaml:19]: For `noarch: python`, restrict the python versions to something like `python >=3.7` both here and under `run`.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/ingeoml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/IngeoML-{{ version }}.tar.gz
+  sha256: 38bcd14a51fa2579e8db81da5bf5bb13f3f53b69c35a59bc47a7d62a7c939c5c
+
+build:
```
[the remark this answers is not in the document]
Comment (mgraffg) [reply recipes/ingeoml/meta.yaml:12]: Yes, you are right. Many thanks!

File: recipes/ingeoml/meta.yaml (hunk 1 above)
Comment (mgraffg) [reply recipes/ingeoml/meta.yaml:20]: I agree. 

PS: It is still hard for me to understand the build time :)

**Comment by synapticarbors** [comment]
I see a commit that was pushed after my comments, but I'm not actually seeing any changes reflected.

**Comment by mgraffg** [comment]
Yes, you are right I made a mistake, made a wrong click.

Sorry for this. I will try to fix this. I will address all your comments.

**Comment by mgraffg** [comment]
Sorry for the mistake. 

Now, I can see the new version, and the changes suggested are set.

Many thanks.

**Review by mgraffg — COMMENTED** [review]
I made a pull request to address these changes.

**Comment by synapticarbors** [comment]
LGTM - I'm happy to merge this when the CI completes

**Pull request merged**